astrocut/cube_cut.py

```diff
@@ -906,17 +902,15 @@ def cube_cut(
             # Get cutout wcs info
             cutout_wcs_full = self._get_full_cutout_wcs(cube[2].header)
             max_dist, sigma = self._fit_cutout_wcs(cutout_wcs_full, img_cutout.shape[1:])
-            if verbose:
-                print("Maximum distance between approximate and true location: {}".format(max_dist))
-                print("Error in approximate WCS (sigma): {}".format(sigma))
+            log.debug("Maximum distance between approximate and true location: %s", max_dist)
+            log.debug("Error in approximate WCS (sigma): %f", sigma)
```

Comment:
When do we format with `%d` or `%f`?

Reply:
I'm using %d for integers, %f for floats, and %s for other types. I added a precision to line 906 in the latest commit.